python/ray/util/sgd/torch/examples/gat_dgl.py

```diff
@@ -0,0 +1,274 @@
+import os
+import torch
+import time
+import numpy as np
+import argparse
+from dgl.dataloading import NodeCollator
+import ray
+from ray.util.sgd import TorchTrainer
+from ray.util.sgd.utils import AverageMeterCollection
+from ray.util.sgd.torch import TrainingOperator
+import dgl
+import torch as th
+import torch.nn as nn
+import torch.nn.functional as F
+from dgl.data import RedditDataset
+from dgl.nn.pytorch import GATConv
+from torch.utils.data import DataLoader
+
+print("Current Path: " + os.getcwd())
+torch.manual_seed(42)
+
+
+# define the model class
+class GAT(nn.Module):
+    def __init__(self, in_feats, n_hidden, n_classes, n_layers, n_heads,
```

Comment:
Could you add a docstring here describing what these args are? It'll help people use the model

Reply:
This case is modified from the original gat case of dgl(https://github.com/dmlc/dgl/blob/master/examples/pytorch/ogb/ogbn-products/gat/main.py). The parameter description can be seen by entering the GATConv class, so there is no need to describe it here.